### PR TITLE
[HttpFoundation] Improved set cookie header tests

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 2.7 (already in master)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Separated from #20569

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Tests/ResponseHeaderBagTest.php [hunk 1]
```diff
@@ -294,4 +293,9 @@ public function provideMakeDispositionFail()
             array('attachment', 'föö.html'),
         );
     }
+
+    protected function assertSetCookieHeader($expected, ResponseHeaderBag $actual)
```
Comment (fabpot) [inline src/Symfony/Component/HttpFoundation/Tests/ResponseHeaderBagTest.php]: Could be private, right?

**Comment by ro0NL** [comment]
@fabpot  changed to private.. this should get in master as well :+1:

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request closed without merging**